eiffel-vocabulary/EiffelAlertAcknowledgedEvent.md

```diff
@@ -141,32 +123,69 @@ __Description:__ The URI of, related to or describing the event sender.
+##### meta.security.integrityProtection.publicKey
 __Type:__ String  
-__Format:__  
-__Required:__ Yes  
-__Description:__ The encrypted digest. The cryptographic hash function and the decryption algorithm to use, similarly to the Trusted Authority (TA), must be known to the recipient. Note that the digest of the entire event is affected by the value of this property. For this reason the input to the hash function SHALL be the entire event unaltered in all parts except for this property, which SHALL be replaced by an empty string.
+__Format:__
+__Required:__ No  
+__Description:__ The producer of the event may include the relevant public key for convenience, rather than relying a separate key distribution mechanism. Note that this property, along with the rest of the event, is encompassed by the integrity protection offered via __meta.security.integrityProtection__.
+
+#### meta.security.sequenceProtection
+__Type:__ Object[]  
+__Format:__
+__Required:__ No  
+__Description:__ An optional object for enabling verification of intact event sequences in a distributed environment, thereby protecting against data loss, race conditions and replay attacks. It allows event publishers to state the order in which they produce a certain set of events. In other words, it cannot provide any global guarantees as to event sequencing, but rather per-publisher guarantees. Every object in the array represents a named sequence of which this event forms a part. For every event including a given named sequence, the publisher SHALL increment __meta.security.sequenceProtection.position__ by 1. The first event produced in a given named sequence SHALL numbered 1.
+
+##### meta.security.sequenceProtection.sequenceName
+__Type:__ String  
+__Format:__
+__Required:__ Yes
+__Description:__ The name of the sequence. There MUST not be two identical __meta.security.sequenceProtection.sequenceName__ values in the same event.
+
+##### meta.security.sequenceProtection.position
+__Type:__ Integer
+__Format:__
+__Required:__ Yes
+__Description:__ The number of the event within the named sequence.
+
 
 ## Version History
-| Version   | Introduced in                                          | Changes                                 |
-| --------- | ------------------------------------------------------ | --------------------------------------- |
-| 1.0.0     |                                                        | Initial version.                        |
+| Version   | Introduced in                                          | Changes                                                                                        |
+| --------- | ------------------------------------------------------ | -----------------------------------------------------------------------------------------------|
+| 2.0.0     | [edition-agen](../../../tree/edition-agen)             | Introduced purl identifiers & Improved information integrity protection                        |
```

Comment:
shouldn't the version be 3.0.0 to reflect the agen protocol?

Reply:
Here version uplifted from 1.0.0, and version 2.0.0 is Agen version for eiffel-operations.

Reply:
True, but it will create confusion as in other events it will be 3.0.0. and since we are manually updating these events will it not be good to use 3.0.0 ?

Reply:
eiffel events may contain each event its own version. I think no problem with 2.0.0

Reply:
I agree with @xmairaa . The versioning is unique per event and it has no explicit connection to any edition. An edition is just a grouping of a decent baseline of individual event versions. Some events in Agen core protocol actually are on 4.0.0 already.